src/components/experiment-tracking/experiment-primary-toolbar/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PrimaryToolbar from '../../primary-toolbar';
+
+export const ExperimentPrimaryToolbar = ({
+  sidebarVisible,
+  setSidebarVisible,
+}) => {
+  return (
+    <PrimaryToolbar
+      visible={{ sidebar: sidebarVisible }}
+      onToggleSidebar={setSidebarVisible}
```

Comment:
Can we try and sort our props alphabetically? It makes things easier to read if this convention is followed throughout the codebase. 

There are other instances of where this can be done in this PR, too, but I'll only write it here once.

Reply:
This is out of context but great suggestion regardless - this isn't a practice we follow in the codebase but this would be a good suggestion to be brought up in the future team norms / coding best practice discussion? We can then look into setting a PR to tackle this and enforcing this in future PR reviews after the discussion.